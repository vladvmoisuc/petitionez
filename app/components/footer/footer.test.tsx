import { render } from "@testing-library/react";

import Footer, { links } from ".";

describe("<Footer />", () => {
  it("should match the snapshot", () => {
    const { container } = render(<Footer />);

    expect(container).toMatchSnapshot();
  });
});

describe("links", () => {
  it("should return the list of stylesheets", () => {
    expect(links()).toEqual([{ href: {}, rel: "stylesheet" }]);
  });
});
