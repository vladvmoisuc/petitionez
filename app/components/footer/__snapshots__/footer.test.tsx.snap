// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match the snapshot 1`] = `
<div>
  <div
    class="Footer vertically-centered  MuiBox-root css-0"
  >
    <h6
      class="MuiTypography-root MuiTypography-subtitle2 MuiTypography-gutterBottom css-11cn0eh-MuiTypography-root"
    >
      Creat de 
      <a
        href="https://www.linkedin.com/in/vladmoisuc/"
        rel="noreferrer"
        target="_blank"
      >
        un bÄƒiat.
      </a>
    </h6>
  </div>
</div>
`;
